vernac/vernacprop.ml

```diff
@@ -0,0 +1,53 @@
+(************************************************************************)
```

Comment:
vernac_classifier is in the same position, it belongs to vernac given how it is named, bet needs to be accessible by Stm.  Maybe the vernac/ directory deserves a better name.  It contain the main interpreter.... I'm all ears